### DISCUSSION TITLE
Diffnow for Namespace org is not working.

### DISCUSSION THREAD

**Discussion by sandeepsupehia** [forcedotcom/salesforcedx-vscode#5783] [Ideas]
### Summary

Diffnow for Namespace org is not working.  I tried with non- namespace org and everything is working fine
### Steps To Reproduce:

1.  Fetch the project in vs from the namespace org (My org also includes Manage Package)
2.  Run the 'SFDX: Diff file against org' for any apex class (also with other files). 

### Expected result
A comparison was expected.
### Actual result

Unexpected error while running ApexProcessor. Error: <classname>.cls was not found in org when querying with username <orgname>

### Additional information

_Feel free to attach a screenshot_.
![image](https://user-images.githubusercontent.com/63860559/87858139-630fd900-c949-11ea-8b23-4015dfcc2445.png)

**VS Code Version**: 1.47.2

**SFDX CLI Version**:7.63.0-c897e8bb1a win32-x64 node-v10.20.1

**OS and version**: windows 10

**Comment by smaddox-sf** [comment]
@sandeepsupehia - Thanks for logging this.  We'll be coming back to the Diff later this year to investigate it and will investigate this gap at that time as well.
